README.md

```diff
@@ -5,15 +5,29 @@ description: >-
 ---
 
 # Introduction
+## OpenLab
 
-## Submitting Your Publication
+OpenLab is a community-led program to test and improve support for the cloud-related SDKs/Tools as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.
 
```

Comment:
I think if we can add a step drawing at here to describe the order of `To be an OpenLab member`, `Test request`, `Development` and `Resource sponsoring`, it is better.